DerekLinan/react-challenge-project-jan-2022
# Dl 4 bug order form user

## Changes
- update payload to proper user value

## Purpose
Allows users to see the user email for each order placed.

## Approach
This PR fixes a bug where the user isn't properly attributed to the order they place.

## Testing Steps
1. Log in with an email and password
2. Place an order on the "Order Form" tab
3. Observe that your email is listed for your order in the "View Orders" tab

Closes [#4](https://github.com/Shift3/react-challenge-project-jan-2022/issues/4)
